lib/tasks/post-build.js

```diff
@@ -1,14 +1,15 @@
 'use strict';
 
-var runCommand = require('../utils/run-command');
-var path       = require('path');
-var chalk      = require('chalk');
-var ui         = require('../ui');
-var Promise    = require('../ext/promise');
+var runCommand      = require('../utils/run-command');
+var defaultPlatform = require('../utils/default-platform');
+var path            = require('path');
+var chalk           = require('chalk');
+var ui              = require('../ui');
+var Promise         = require('../ext/promise');
```

Comment:
'Promise' is defined in a future version of JavaScript. Use a different variable name to avoid migration issues.<br>'require' is not defined.